pynucastro/rates/library.py

```diff
@@ -357,15 +445,33 @@ def __add__(self, other):
         return new_library
 
     def __sub__(self, other):
-        """Return a Library containing the rates in this library that are not
-        contained in other_library"""
+        """Return a Library containing the rates in this library that
+        are not contained in other_library
+
+        """
 
         diff_rates = set(self.get_rates()) - set(other.get_rates())
         new_library = Library(rates=diff_rates)
         return new_library
 
     def get_rate_by_nuclei(self, reactants, products):
-        """given a list of reactants and products, return any matching rates"""
+        """Given a list of reactants and products, return any matching
+        rates
+
+        Parameters
+        ----------
+        reactants : list of Nucleus or str
+            the list of nuclei that serve as reactants.
+        products : list of Nucleus or str
+            the list of nuclei that serve as products.
+
+        Returns
+        -------
+        Rate, list(Rate)
+            a list of nuclei or a single nucleus (if there is only one)
```

Comment:
here too, `a list of rates or a single rate`

Reply:
fixed